libs/service-portal/vehicles/src/components/DetailTable/AxleTable.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { Box, Table as T, Text } from '@island.is/island-ui/core'
+import { Tyres, VehiclesAxle } from '@island.is/api/schema'
+import { messages } from '../../lib/messages'
+import { displayWithUnit } from '../../utils/displayWithUnit'
+
+interface PropTypes {
+  axles?: VehiclesAxle[] | null
+  tyres?: Tyres | null
+}
+
+const AxleTable = ({ axles, tyres }: PropTypes) => {
+  useNamespaces('sp.vehicles')
+  const { formatMessage } = useLocale()
+
+  const tyreArray: string[] = []
+  tyres?.axle1 && tyreArray.push(tyres?.axle1)
+  tyres?.axle2 && tyreArray.push(tyres?.axle2)
+  tyres?.axle3 && tyreArray.push(tyres?.axle3)
+  tyres?.axle4 && tyreArray.push(tyres?.axle4)
+  tyres?.axle5 && tyreArray.push(tyres?.axle5)
+
+  return (
+    <Box marginBottom={4} marginTop="containerGutter">
+      <T.Table>
+        <T.Head>
+          <T.Row>
+            <T.HeadData>
+              <Text variant="medium" fontWeight="semiBold">
+                {formatMessage(messages.axle)}
+              </Text>
+            </T.HeadData>
+            <T.HeadData>
+              <Text variant="medium" fontWeight="semiBold">
+                {formatMessage(messages.axleTotalWeight)}
+              </Text>
+            </T.HeadData>
+            <T.HeadData>
+              <Text variant="medium" fontWeight="semiBold">
+                {formatMessage(messages.axleWheel)}
+              </Text>
+            </T.HeadData>
+          </T.Row>
+        </T.Head>
+        <T.Body>
+          {axles?.map((item: VehiclesAxle, index) => {
```

Comment:
Should this map the `tyreArray` or is it a case of where the axles will always match the tyreArray?

Reply:
Yes axles and tyre size will always be hand in hand. I changed it a little to prevent null errors, put empty string if no value in tyre array so the table will still be rendered correctly.